src/schema/v2/me/conversation/eventConnection/hybridConnection/README.md

```diff
@@ -0,0 +1,72 @@
+# Hybrid connection
+
+_Tools for building a relay connection from multiple sources_
+
+This is a small module that builds a relay connection which is served from multiple sources. It aims to have a similar interface to [`connectionFromArraySlice`](https://github.com/graphql/graphql-relay-js/blob/17231860ee64431009c193be2e6ac444936a88ea/src/connection/arrayConnection.js#L49), which serializes its cursors as `arrayconnection:{index}`, replacing a single index with a serialized (query) string of last-seen indexes for each source collection as well as the overall position.
+
+It exports a single function, `fetchHybridConnection` which accepts a map of keys(arbitrary strings for each source) and values (fetchers which can respect `limit`/`offset` args). The function decodes the `after` cursor if present and fetches from each source using its own separately-tracked offset arguments. Then it attaches new offsets and cursors before returning the expected connection interface.
+
+Example:
+
+```ts
+const result = await fetchHybridConnection(
+  { first: 4 },
+  {
+    // a fetcher for messages
+    msg: fetchMessagesForPagination(
+      conversationID,
+      conversationMessagesLoader,
+      parent
+    ),
+    // a fetcher for order events
+    ord: fetchOrderEventsForPagination(
```

Comment:
`ord` -> `orders` 

(We should strive to avoid abbreviation in code)

Reply:
these keys get serialized in the cursors so I was shooting for brevity, but a different dsl could be possible.

Reply:
lets avoid abbreviating. worrying about a descriptive string being serialized and therefore abbreviating is an over-optimization